tests/testproject/TestProject.AppHost/TestProgram.cs

```diff
@@ -5,12 +5,29 @@
 using System.Text.Json;
 using System.Text.Json.Nodes;
 using Aspire.Hosting.Lifecycle;
+using Aspire.TestProject;
 
 public class TestProgram : IDisposable
 {
     private TestProgram(string[] args, Assembly assembly, bool includeIntegrationServices, bool includeNodeApp, bool disableDashboard)
     {
-        if (args.Contains("--disable-dashboard"))
+        IList<TestResourceNames> resourcesToSkip = [];
```

Comment:
Do we actually need this `resourcesToSkip`? If we just skipped the cosmos and oracle tests, wouldn't that be enough? Yes it would still spin up those containers during the tests, but nothing would try to connect to them.

I don't really like all these if conditions everywhere. It really clutters up the code.

Reply:
We have one test that health checks on the whole thing, and that would continue to fail.

> I don't really like all these if conditions everywhere. It really clutters up the code.
agree!